server/services/userService.js

```diff
@@ -1,6 +1,28 @@
 const allowedRoles = require('../authentication/roles');
 
-module.exports = function(nomisClientBuilder) {
+module.exports = function(logger, nomisClientBuilder) {
+
+    async function getUserProfile(token, refreshToken, username) {
+        logger.info(`User profile for: ${username}`);
+        const nomisClient = nomisClientBuilder(token);
+
+        const [profile, roles] = await Promise.all([
+            nomisClient.getLoggedInUserInfo(),
+            getAllRoles({token})
+        ]);
+
+        logger.info(`User profile success - token: ${token}`);
```

Comment:
don't think it is a good idea to log the user's `token` here

Reply:
Agreed